operations/power_management/Prepare_the_System_for_Power_Off.md

```diff
@@ -37,7 +37,13 @@ An authentication token is required to access the API gateway and to use the `sa
 
     3.  If the Slingshot network includes edge switches, obtain the user ID and password for these switches.
 
-2.  Determine which Boot Orchestration Service \(BOS\) templates to use to shut down compute nodes and UANs. You can list all the session templates using `cray bos v1 sessiontemplate list`. If you are unsure of which template is in use, you can call `sat status` to find the xname, then use `cray cfs components describe XNAME` to find the bos_session, and use `cray bos v1 session describe BOS_SESSION` to find the `templateUuid`. Then finally use `cray bos v1 sessiontemplate describe TEMPLATE_UUID` to determine the list of xnames associated with a given template. For example:
+2.  Use `sat auth` to authenticate to the API gateway within SAT.
+
+    If SAT has already been authenticated to the API gateway, this step may be skipped.
+
+    See the "SAT Authentication" section in the System Admin Toolkit (SAT) product stream documentation (S-8031) for instructions on how to acquire a SAT authentication token.
+
+3.  Determine which Boot Orchestration Service \(BOS\) templates to use to shut down compute nodes and UANs. You can list all the session templates using `cray bos v1 sessiontemplate list`. If you are unsure of which template is in use, you can call `sat status` to find the xname, then use `cray cfs components describe XNAME` to find the bos_session, and use `cray bos v1 session describe BOS_SESSION` to find the `templateUuid`. Then finally use `cray bos v1 sessiontemplate describe TEMPLATE_UUID` to determine the list of xnames associated with a given template. For example:
```

Comment:
Instead of listing all of the steps in a block of text, it would be better to break this down into sub-steps. Something like the following could work better:

3. Determine which Boot Orchestration Service \(BOS\) templates to use to shut down compute nodes and UANs.
    
    There will be a seperate session template for UANs and computes nodes.
    
    1. List all the session templates.
        
        If it is unclear what session template is in use, proceed to the next substep.

        ```
        ncn-m001# cray bos v1 sessiontemplate list
        ```

    2. Find the xname with `sat status`.

        ```bash
        ncn-m001# sat status | grep "Compute\|Application"
        ```

        Example output:

        ```
        | x3000c0s19b1n0 | Node | 1        | On    | OK   | True    | X86  | River | Compute     | Sling    |
        | x3000c0s19b2n0 | Node | 2        | On    | OK   | True    | X86  | River | Compute     | Sling    |
        | x3000c0s19b3n0 | Node | 3        | On    | OK   | True    | X86  | River | Compute     | Sling    |
        | x3000c0s19b4n0 | Node | 4        | On    | OK   | True    | X86  | River | Compute     | Sling    |
        | x3000c0s27b0n0 | Node | 49169248 | On    | OK   | True    | X86  | River | Application | Sling    |
        ```

    3. Find the `bos_session` value via the Configuration Framework Service (CFS).

        ```
        ncn-m001# cray cfs components describe XNAME | grep bos_session
        ```

        Example output:

        ```
        bos_session = "e98cdc5d-3f2d-4fc8-a6e4-1d301d37f52f"
        ```

    4. Find the required `templateUuid` value with BOS.
        
        ```
        ncn-m001# cray bos v1 session describe XNAME | grep templateUuid
        ```

        Example output:

        ```
        templateUuid = "compute-nid1-4-sessiontemplate"
        ```

    5. Determine the list of xnames associated with the desired boot session template.

        ```
        ncn-m001# cray bos v1 sessiontemplate describe SESSION_TEMPLATE_NAME | grep node_list
        ```

        Example output:

        ```
        node_list = [ "x3000c0s19b1n0", "x3000c0s19b2n0", "x3000c0s19b3n0", "x3000c0s19b4n0",]
        ```